Modules/System/Cryptography Management/src/DESCryptoServiceProvider.Codeunit.al

```diff
@@ -47,23 +47,25 @@ codeunit 1379 DESCryptoServiceProvider
     /// Encrypts data in stream with DotNet Cryptography.DESCryptoServiceProvider
     /// </summary>
     /// <param name="Password">Represents the password to be used to initialize a new instance of Rfc2898DeriveBytes</param>
+    /// <param name="Salt">Represents the salt to be used to initialize a new instance of System.Security.Cryptography.Rfc2898DeriveBytes</param>
     /// <param name="InputInstream">Represents the input instream data to encrypt</param>
     /// <param name="OutputOutstream">Represents the output instream encrypted data</param>
     [NonDebuggable]
-    procedure EncryptStream(Password: Text; InputInstream: InStream; var OutputOutstream: Outstream)
+    procedure EncryptStream(Password: Text; Salt: Text; InputInstream: InStream; var OutputOutstream: Outstream)
```

Comment:
You are changing public function signatures, by adding the salt parameter. That would be a breaking change, which is no longer allowed. Please add function overloads instead. If the functions with the old signature aren't useful any longer, please deprecate those.

Reply:
Fair enough, they aren't useful like this, so I made them obsolete.